apps/happy-blocks/block-library/support-content-footer/index.php

```diff
@@ -97,7 +97,7 @@
 			</p>
 		</div>
 		<div class="support-content-subscribe">
-			<p><?php esc_html_e( 'Get the latest learning in your inbox::', 'happy-blocks' ); ?></p>
+			<p><?php esc_html_e( 'Get the latest learning in your inbox:', 'happy-blocks' ); ?></p>
```

Comment:
I am uncertain if the double `::` is intentional, 
<img width="404" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/e5d6c9fc-8609-4344-a503-3430f13cfd68">